sklearn/feature_extraction/text.py

```diff
@@ -328,7 +328,7 @@ def _validate_custom_analyzer(self):
             pass
 
     def build_analyzer(self):
-        """Return a callable that handles preprocessing and tokenization"""
+        """Return a callable that handles preprocessing, tokenization and n-grams generation"""
```

Comment:
Please keep lines under 80 characters

Reply:
Also from https://www.python.org/dev/peps/pep-0257/

> Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; it is important that it fits on one line and is separated from the rest of the docstring by a blank line.

So we can't add a newline break here ether. The only solution could be to remove some words from the summary line.